### ISSUE TITLE
# NOQA is not honoured

### ISSUE THREAD

**Issue by haizaar** [timothycrosley/isort#679]
```
$ cat main.py 

import os

import requests
import ujson  # NOQA

import foo

$ touch foo.py
$ isort --diff main.py 
--- /tmp/bar/main.py:before	2018-02-20 15:58:49.566743
+++ /tmp/bar/main.py:after	2018-02-20 16:00:04.680267
@@ -2,6 +2,6 @@
 import os
 
 import requests
-import ujson  # NOQA
 
 import foo
+import ujson  # NOQA
```
I.e. I would expect the NOQA line not to be touched at all.

The system info:
```
$ cat /etc/lsb-release 
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=16.04
DISTRIB_CODENAME=xenial
DISTRIB_DESCRIPTION="Ubuntu 16.04.3 LTS"
$ python --version
Python 3.6.4
$ pip freeze
certifi==2018.1.18
chardet==3.0.4
idna==2.6
isort==4.3.4
requests==2.18.4
ujson==1.35
urllib3==1.22
```

Thanks!

**Comment by luislew** [comment]
Seeing the same issue:
```
 import operator
 
-from elasticsearch_dsl.query import Bool, Exists as ESExists, Match, Range, Script, Terms  # isort:skip
 from schematics.exceptions import ValidationError
 
 from nw_api.exceptions import InvalidInputError
@@ -11,6 +10,8 @@ from marketplace.canonical_objects.types import AttributePathType, BaseType, Str
 from marketplace.exceptions import FilterRuntimeError
 from marketplace.schema.field_type import FieldTypeCategories
 
+from elasticsearch_dsl.query import Bool, Exists as ESExists, Match, Range, Script, Terms  # isort:skip
```

**Comment by richardlthomas** [comment]
Fixed in PR #843
